apps/lib/p5play/p5.play.js

```diff
@@ -1482,14 +1484,16 @@ function Sprite(pInst, _x, _y, _w, _h) {
       throw new TypeError('setCollider expects the first argument to be either "circle" or "rectangle"');
     } else if (type === 'circle' && !(arguments.length === 1 || arguments.length === 4)) {
       throw new TypeError('Usage: setCollider("circle") or setCollider("circle", offsetX, offsetY, radius)');
-    } else if (type === 'rectangle' && !(arguments.length === 5)) {
-      throw new TypeError('Usage: setCollider("rectangle", offsetX, offsetY, width, height)');
+    } else if (type === 'rectangle' && !(arguments.length === 1 || arguments.length === 5)) {
+      throw new TypeError('Usage: setCollider("rectangle") or setCollider("rectangle", offsetX, offsetY, width, height)');
     }
 
     this.colliderType = 'custom';
 
     var v = createVector(offsetX, offsetY);
-    if (type === 'rectangle' && arguments.length === 5) {
+    if (type === 'rectangle' && arguments.length === 1) {
+      this.collider = new AABB(pInst, this.position, createVector(this.width, this.height));
```

Comment:
It is; [it defaults to a (0, 0) vector](https://github.com/code-dot-org/code-dot-org/blob/525320d1e4a1ea46f23d8378b53032f0759c99f2/apps/lib/p5play/p5.play.js#L3073) if omitted.